docs/jwst/outlier_detection/outlier_detection_coron.rst

```diff
@@ -25,9 +24,6 @@ Specifically, this routine performs the following operations:
 #. Perform statistical comparison between median image and original image to identify outliers.
 
    The core detection algorithm uses the following to generate an outlier mask
-
    .. math:: | image\_input - image\_median | > SNR*input\_err
 
 #. Update input data model DQ arrays with mask of detected outliers.
```

Comment:
SCI, ERR, and VAR* arrays are updated too.

Reply:
These are set to NaN at those locations now, right?

Reply:
Yes, that's right.